src/components/datagrid/body/data_grid_body.tsx

```diff
@@ -271,6 +271,32 @@ InnerElement.displayName = 'EuiDataGridInnerElement';
 const INITIAL_ROW_HEIGHT = 34;
 const IS_JEST_ENVIRONMENT = global.hasOwnProperty('_isJest');
 
+/**
+ * getParentCellContent is called by the grid body's mutation observer,
+ * which exists to pick up DOM changes in cells and remove interactive elemnts
```

Comment:
small nit, not a blocker
```suggestion
 * which exists to pick up DOM changes in cells and remove interactive elements
```